addons/big-save-button/addon.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Larger \"Save Now\" button",
+  "description": "Extends the size of the \"Save Now\" button in the editor, making it easier to click.",
+  "tags": ["editor", "editorMenuBar", "recommended"],
+  "versionAdded": "1.36.0",
+  "enabledByDefault": true,
```

Comment:
!! just a reminder

Reply:
That it's enabled by default?

Reply:
yep

Reply:
the relevant discussion is in the issue
just wanted to point out that it's intentional